lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntime.cpp

```diff
@@ -462,6 +465,7 @@ SwiftLanguageRuntimeImpl::GetSwiftMetadataCache() {
 }
 
 void SwiftLanguageRuntimeImpl::SetupReflection() {
+  // SetupReflection MUST be locked before being called.
```

Comment:
Any way we can assert this?

Reply:
I looked this up when adding this comment and unfortunately not...